force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +389,18 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
+        final Decimal maxMoney = 40;
+        Decimal moneyCounter = 0;
+
+        for (Decimal amount : moneyInWallet) {
+            moneyCounter += amount;
+            if (moneyCounter > maxMoney) {
+                break;
```

Comment:
Well structured placement of the break. I also like that you used a constant for maxMoney!